inspirehep/dojson/hep/model.py

```diff
@@ -26,4 +26,19 @@
 from dojson import utils
 
 hep = Overdo()
+
+
+@hep.over('book', '^773..')
```

Comment:
Looking at the `dojson` source code I don't think this ensures that this code will always be run at the end (after the collections have been populated)

With a bit of Python magic, I have created this PR (https://github.com/inspirehep/inspire-next/pull/666) to show another way that would ensure we execute the code at the end.

@kaplun @jalavik what do you think? This is in order to allow for some code to execute after dojson is finished (to ensure all the keys are present)